### ISSUE TITLE
jsonlog.logger - error occurred creating the configuration section handler

### ISSUE THREAD

**Issue by Surfrat** [mperdeck/jsnlog#155]
Hi,

I have updated all my Nuget packages and something has broken the Jsnlog logging. My backend logging continues to work but not my logging from my .js files.

I get this error:
![image](https://user-images.githubusercontent.com/2066487/32730788-d346ef9a-c890-11e7-97e7-5eb5ab1697a9.png)

I noticed this error in an Application Insights log:

> "message": "Could not load file or assembly 'Common.Logging.Log4Net.Universal' or one of its dependencies. The system cannot find the file specified.",

but it is in my packages folder:
![image](https://user-images.githubusercontent.com/2066487/32731766-5f1abc20-c893-11e7-8154-8cb5fd3f7564.png)

[Logs.zip](https://github.com/mperdeck/Common.Logging.Log4Net.Universal/files/1467284/Logs.zip)

The full error message, packages.config and the web.config are attached.

I have spent half a day researching this and I have not found a solution. Your help will be appreciated.

Thanks.

**Comment by Surfrat** [comment]
Problem is caused by Common.Logging.Log4Net.Universal.dll not being copied to the Bin folder. I manually copy it and it worked.

Yet to find a fix though.

**Comment by Surfrat** [comment]
I found the problem. 

My web application uses .Net 4.5.2.

The nuget Common.Logging.Log4Net.Universal package created a structure as follows:
![image](https://user-images.githubusercontent.com/2066487/32886191-9501493a-cac8-11e7-9157-43795f625e53.png)

I added net40 folder and copied the dll accross. This gave me this:
![image](https://user-images.githubusercontent.com/2066487/32886239-bca03f82-cac8-11e7-8b2f-3915f2980dde.png)

I also edited my csproj file to point to the net40 folder.
`<Reference Include="Common.Logging.Log4Net.Universal, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
      <HintPath>..\packages\Common.Logging.Log4Net.Universal.1.2.0\lib\net40\Common.Logging.Log4Net.Universal.dll</HintPath>
    </Reference>`

This has fixed it. Am I correct is assuming the Nuget package needs a net40 folder?
